orch-configs/clusters/dev-minimal.yaml

```diff
@@ -16,7 +16,6 @@ root:
     - orch-configs/profiles/proxy-none.yaml
     - orch-configs/profiles/profile-dev.yaml
     - orch-configs/profiles/artifact-rs-production-noauth.yaml
```

Comment:
Since it is removed all the places, then should we remove the original yaml file orch-configs/profiles/enable-osrm-manual-mode.yaml?